supervisor/http.py

```diff
@@ -663,7 +659,7 @@ def more(self):
             self.file.seek(-bytes_added, 2)
             bytes = self.file.read(bytes_added)
             self.sz = newsz
-            return bytes
+            return as_string(bytes)
```

Comment:
@regebro: the other changes are aimed to display unicode log. Please help to review them too!